### PR TITLE
Implementing create service account flow (s16)

### PR DESCRIPTION
**Pull request by tahmidefaz**

### CODE REVIEW

File: actions/platform/service_account/create_service_account.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+NEW_OR_EXISTING = "new_or_existing"
+SERVICE_ACCOUNT_NAME = "service_account_name"
+SERVICE_ACCOUNT_DESCRIPTION = "service_account_description"
+CREATE_ANOTHER_SERVICE_ACCOUNT = "create_another_service_account"
+
+new_or_existing_choice = FuzzySlotMatch(
+    "new_or_existing",
+    [
+        FuzzySlotMatchOption("new"),
```
Comment (justinorringer) [inline actions/platform/service_account/create_service_account.py:23]: What if they say "one I already have" just to trip us up haha. Mostly kidding, but it doesn't seem robust, but maybe thats all good.

File: data/service_accounts/domain.yml [hunk 2]
```diff
@@ -47,16 +48,65 @@ slots:
         intent: intent_core_no
         conditions: *need_another_integration
 
+  new_or_existing:
```
Comment (justinorringer) [inline data/service_accounts/domain.yml:51]: We've been scoping these var and intents to avoid conflicts, like service_account_new_or_existing.

The names do get kind of long, maybe could abbreviate for all of these. Either way I think it'd be helpful, could do in another PR if thatd be easier.

**Review by justinorringer — APPROVED** [review]
Looking good! A couple little things

File: data/service_accounts/domain.yml (hunk 2 above)
Comment (tahmidefaz) [reply data/service_accounts/domain.yml]: Just renamed this slot only. I other ones have `service_account` in it already.

File: actions/platform/service_account/create_service_account.py (hunk 1 above)
Comment (tahmidefaz) [reply actions/platform/service_account/create_service_account.py:23]: Yeah, you can potentially say a lot of things like this under `new`. I am skeptical if we are able to capture all the edge case for this one. We can add more in the QE/Feedback stage if necessary. Not making any changes for this one yet.
